src/language-css/parser-postcss.js

```diff
@@ -92,29 +97,70 @@ function parseValueNodes(nodes) {
   return rootParenGroup;
 }
 
-function stringifyGroup(node) {
-  if (node.group) {
-    return stringifyGroup(node.group);
-  }
-
+function stringifyNode(node) {
   if (node.groups) {
-    return node.groups.reduce((previousValue, currentValue, index) => {
+    const open = (node.open && node.open.value) || "";
+    const groups = node.groups.reduce((previousValue, currentValue, index) => {
       return (
         previousValue +
-        stringifyGroup(currentValue) +
-        (currentValue.type === "comma_group" && index !== node.groups.length - 1
+        stringifyNode(currentValue) +
+        (node.groups[0].type === "comma_group" &&
+        index !== node.groups.length - 1
           ? ","
           : "")
       );
     }, "");
+    const close = (node.close && node.close.value) || "";
+
+    return open + groups + close;
   }
 
-  const before = node.raws && node.raws.before ? node.raws.before : "";
-  const value = node.value ? node.value : "";
-  const unit = node.unit ? node.unit : "";
-  const after = node.raws && node.raws.after ? node.raws.after : "";
+  const before = (node.raws && node.raws.before) || "";
+  const quote = (node.raws && node.raws.quote) || "";
+  const atword = node.type === "atword" ? "@" : "";
+  const value = node.value || "";
+  const unit = node.unit || "";
+  const group = node.group ? stringifyNode(node.group) : "";
+  const after = (node.raws && node.raws.after) || "";
+
+  return before + quote + atword + value + quote + unit + group + after;
+}
+
+function groupListContainsSassInterpolationMarker(groups) {
+  if (groups && groups.length) {
+    for (let i = groups.length - 1; i > 0; i--) {
+      // If we find `#{`, return true.
+      if (
+        groups[i].type === "word" &&
+        groups[i].value === "{" &&
+        groups[i - 1].type === "word" &&
+        groups[i - 1].value.endsWith("#")
+      ) {
+        return true;
+      }
+    }
+  }
+  return false;
+}
```

Comment:
Please move that to utils and rename to `hasScssInterpolation`

Reply:
Moved to utils and renamed to `hasSCSSInterpolation` (to match local convention).